addon/templates/components/truncate-multiline.hbs

```diff
@@ -1,18 +1,21 @@
 {{#if truncate}}
   <div class="truncate-multiline--truncation-target">
     {{~#if hasBlock~}}
-      {{yield}}
+      {{~yield~}}
```

Comment:
These tildes are unnecessary b/c of the surrounding tildes. Tilde removes whitespace up to the next non-whitespace thing, which means back-to-back tildes are redundant.